playbooks/stopMasterWorker.yml

```diff
@@ -8,8 +8,6 @@
     become_user: ubuntu
```

Comment:
this needs `become=true` also?

Reply:
I don't think so, just the workers needs to be started as root